NineMensMorris/src/game.cpp

```diff
@@ -165,13 +182,28 @@ void Game::spaceCleanup(std::vector<Space*> &spaces){
     spaces.clear();
 }
 
+//Freeing the button memory at the end of game
+void Game::buttonCleanup() {
+    scene->removeItem(forfeitButton->graphicsProxyWidget());
+    scene->removeItem(menuButton->graphicsProxyWidget());
+    scene->removeItem(playAgainButton->graphicsProxyWidget());
+    delete forfeitButton;
+    delete menuButton;
+    delete playAgainButton;
+}
+
 void Game::textItemCleanup() {
 /* Remove text items from memory at the end of the game */
     scene->removeItem(titleText);
     scene->removeItem(instructionText);
     scene->removeItem(turnText);
     scene->removeItem(whitePieceText);
     scene->removeItem(blackPieceText);
+    delete blackPieceText;
+    delete whitePieceText;
+    delete titleText;
+    delete instructionText;
+    delete turnText;
```

Comment:
Thank you for correcting this method. I saw Ryan's comment on your earlier PR about how we weren't cleaning up the QGraphicsProxyWidgets properly and this implementation popped in my mind.